epoxy-processor/src/main/java/com/airbnb/epoxy/ControllerModelField.kt

```diff
@@ -0,0 +1,36 @@
+package com.airbnb.epoxy
+
+import com.squareup.javapoet.TypeName
+
+import javax.lang.model.element.Element
+
+import com.airbnb.epoxy.Utils.isFieldPackagePrivate
+
+class ControllerModelField(element: Element) {
+
+    var fieldName: String = element.simpleName.toString()
+    var typeName: TypeName = TypeName.get(element.asType())
+    var packagePrivate: Boolean = isFieldPackagePrivate(element)
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
Yep, you are right. I haven't thought about it but it does look like a good approach. I'll fix it in upcoming PR.